src/views/AdminActivityEdit.vue

```diff
@@ -153,19 +315,36 @@ export default {
     ...mapGetters(["activities", "currentCountry"]),
     activity() {
       return this.activities[this.currentKey];
+    },
+    keywords() {
+      return this.activity.activityKeywords ? this.activity.activityKeywords.split(",") : [];
+    },
+    hasBetaAccess() {
+      return !!this.currentUserSettings?.hasBetaAccess;
     }
   },
   methods: {
     saveField(name, event) {
       console.log("So... you want to save...", { name, event });
       console.log("New value", event.target.value);
+      console.log("event", event);
       db.ref("content")
         .child(this.currentCountry)
         .child("activities")
         .child(this.currentKey)
         .child(name)
         .set(event.target.value);
     },
+    saveValue(name, value,event) {
```

Comment:
I don't understand saveField vs saveValue -- they're both being passed `$event` as their second param?

Reply:
saveField is going away after we finalize the move to phase2.

in a nutshell, vuetify controls save the actual value that you changed it to to $event whereas the older html controls that existed previously does not. saveField needed to use event.target.value where as saveValue could use the parameter directly.

Reply:
ahh, very good